mobile_back_pixel/docker/mlperf_mobile/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:bionic
```

Comment:
File needs a copyright header

Reply:
Done.

Reply:
Why do we need this file? We have Dockerfile with the same image name here https://github.com/mlcommons/mobile_app_open/tree/master/android/docker/mlperf_mobile

Reply:
removed via f1dc6f3